File: rust-port/testdata/capabilities/Probe.cs
using Example.Helpers;

namespace Probe
{
    public class Widget : BaseWidget, IRenderable
    {
        private string name;

        public string Render()
        {
            return Helper.Help(this.name);
        }

        public static void Main()
        {
            var w = new Widget();
            w.Render();
        }
    }
}

File: rust/testdata/capabilities/probe.c
#include <stdio.h>
#include "helper.h"

struct Widget {
    const char *name;
};

const char *widget_render(struct Widget *w) {
    return helper_help(w->name);
}

int main(void) {
    struct Widget w;
    widget_render(&w);
    return 0;
}
